app/src/main/java/org/simple/clinic/summary/PatientSummaryModule.kt

```diff
@@ -11,6 +11,6 @@ class PatientSummaryModule {
   @Provides
   fun providesSummaryConfig(): Single<PatientSummaryConfig> = Single.just(PatientSummaryConfig(
       numberOfBpPlaceholders = 3,
-      bpEditableFor = Duration.ofMinutes(1L)
+      bpEditableFor = Duration.ofHours(1L)
```

Comment:
I personally find adding the type of a field to the name unnecessary in obvious cases. 

When we speak naturally, we will still say "The BP is editable for <X minutes>". The "X minutes" part here is encoded in the type `Duration` which the IDE and type system will take care of for you.

Reply:
@sanchita I don't think that suffixing a variable with its type adds more meaning to the variable. It's useful in some cases to describe what type of values does a variable holds. that way, we would start naming variables like `systolicInt`, `patientNameString`, and so on. :)

Reply:
@sanchita I don't think that suffixing a variable with its type adds more meaning to the variable. It's useful in some cases to describe what type of values does a variable holds. Otherwise, we will start naming variables like `systolicInt`, `patientNameString`, and so on. :)